marklogic-data-hub-central/ui/src/components/entities/entity-tiles.test.tsx

```diff
@@ -142,13 +142,15 @@ describe("Entity Tiles component", () => {
       await fireEvent.click(customerPanel);
       // Check for Mapping tab
       expect(getByText('Map')).toBeInTheDocument();
-      expect(getByText('Custom')).toBeInTheDocument();
+      await fireEvent.click(getByText('Custom'));
+      expect(getByText("You can create Custom steps either manually or using Gradle, then deploy them. Deployed Custom steps appear here. Hub Central only allows running Custom steps, not editing or deleting them.")).toBeInTheDocument();
```

Comment:
I prefer not explicitly testing long strings of text and instead labeling those passages with an aria-label and testing for that. That way we can make minor changes to the text content without having to worry about the tests. I recommend adding:

aria-label={'customEntityTitle'}

and then check like so:

expect(getByLabelText('customEntityTitle')).toBeInTheDocument();

Reply:
Updated!!